src/microbe_stage/MicrobeAI.cs

```diff
@@ -774,7 +774,7 @@ private void LowerPursuitThreshold()
 
     private bool CanTryToEatMicrobe(Microbe targetMicrobe)
     {
-        var sizeRatio = microbe.EngulfSize / targetMicrobe.EngulfSize;
+        var sizeRatio = microbe.Size / targetMicrobe.Size;
```

Comment:
Why is EngulfSize no longer used?

Reply:
Well it's replaced with Size, since EngulfSize is just simply the cell size I figured it would make sense to generalize the property name.

Reply:
I guess we have basically decided that size is the engulf size. I'm just thinking that perhaps we sometime would want to have a different size than engulf size, but maybe we won't. So I guess if you also can't foresee that happening, let's go with this change.

Reply:
Yeah I don't see that changing for the time being.